source/products/zurich/zurich-do/index.rst

```diff
@@ -1,4 +1,4 @@
-Zurich D&O
+D&O
```

Comment:
Looks weird just d&o. But I don't think we should change it, cause we internally get what that is already 😆

Reply:
Change that because of duplicate text in docs.
This is how it looks right now:
![Screenshot 2020-02-04 at 12 10 10](https://user-images.githubusercontent.com/4901997/73735039-4d19ad80-4747-11ea-86ea-c985ee5d1911.png)

And for example `Roland` products using without `Roland`:
![Screenshot 2020-02-04 at 12 10 43](https://user-images.githubusercontent.com/4901997/73735083-602c7d80-4747-11ea-8048-a73d39c7cbb2.png)

Don't know which way we should go, but it should be same for all products. 😄

Reply:
Agree. Looks better the way you changed